app.test.js

```diff
@@ -20,3 +37,51 @@ describe('Get health route', () => {
     expect(res.statusCode).toBe(404);
   });
 });
+
+const testSoldier = {
```

Comment:
same problem as `app.js`. it will become too complex and  huge to handle...

General advice: use the same project structure in your code and your test, for example:
- `app.js` (or `src/app.js`) -> `test/app.test.js`
- `src/utils/boris-be-nice.js` -> `test/utils/boris-be-nice.test.js`

Reply:
fixed